[AWS-IAM Identity center] enhancement (#33850)
demisto/content: Packs/AWS-IAMIdentityCenter/ReleaseNotes/1_0_2.md

#### Integrations

##### AWS - IAM Identity Center

- Updated the context path prefix to be `AWS.IAMIdentityCenter` instead of `AWS.IAM`.
- Created a new playbook.
- Added 6 commands:
    - ***aws-iam-identitycenter-delete-user***
    - ***aws-iam-identitycenter-create-group***
    - ***aws-iam-identitycenter-delete-group***
    - ***aws-iam-identitycenter-list-memberships***
    - ***aws-iam-identitycenter-update-user***
    - ***aws-iam-identitycenter-update-group***
- Added support for the following arguments in the ***aws-iam-identitycenter-create-user***, ***aws-iam-identitycenter-list-users***, ***aws-iam-identitycenter-get-user***, ***aws-iam-identitycenter-get-user-by-email***, ***aws-iam-identitycenter-list-groups***,***aws-iam-identitycenter-get-group***, ***aws-iam-identitycenter-add-user-to-group***, ***aws-iam-identitycenter-list-groups-for-user*** and ***aws-iam-identitycenter-delete-group-membership*** commands.
    - *identityStoreId*
    - *region*
    - *roleArn*
    - *roleSessionDuration*
    - *roleSessionName*

> - Added the full API responses to the context data in the ***aws-iam-identitycenter-create-user***, ***aws-iam-identitycenter-list-users***,  ***aws-iam-identitycenter-get-user***, ***aws-iam-identitycenter-get-user-by-email***, ***aws-iam-identitycenter-list-groups***, ***aws-iam-identitycenter-get-group*** and ***aws-iam-identitycenter-list-groups-for-user*** commands.

Reviewer: ```suggestion
- Added the full API responses to the context data in the following commands:
   - ***aws-iam-identitycenter-create-user***
   - ***aws-iam-identitycenter-list-users***
   - ***aws-iam-identitycenter-get-user***
   - ***aws-iam-identitycenter-get-user-by-email***
   - ***aws-iam-identitycenter-list-groups***
   - ***aws-iam-identitycenter-get-group***
   - ***aws-iam-identitycenter-list-groups-for-user***
```

> - Updated the context path to be `AWS.IAMIdentityCenter.User` in the ***aws-iam-identitycenter-create-user***, ***aws-iam-identitycenter-list-users***,  ***aws-iam-identitycenter-get-user***, ***aws-iam-identitycenter-get-user-by-email*** and ***aws-iam-identitycenter-list-groups-for-user*** commands.

Reviewer: ```suggestion
- Updated the context path to be `AWS.IAMIdentityCenter.User` in the following commands:
   - ***aws-iam-identitycenter-create-user***
   - ***aws-iam-identitycenter-list-users***
   - ***aws-iam-identitycenter-get-user***
   - ***aws-iam-identitycenter-get-user-by-email***
   - ***aws-iam-identitycenter-list-groups-for-user*** 
```

> - Updated the context path to be `AWS.IAMIdentityCenter.Group` in the ***aws-iam-identitycenter-list-groups*** and ***aws-iam-identitycenter-get-group*** commands.

Reviewer: ```suggestion
- Updated the context path to be `AWS.IAMIdentityCenter.Group` in the following commands:
   - ***aws-iam-identitycenter-list-groups***
   - ***aws-iam-identitycenter-get-group***
```

> - Deprecated the ResponseMetadata from context data and human readable in ***aws-iam-identitycenter-create-user***, ***aws-iam-identitycenter-get-user***, ***aws-iam-identitycenter-create-group*** and ***aws-iam-identitycenter-get-group*** commands.

Reviewer: ```suggestion
- Deprecated the ResponseMetadata from context data and human readable in the following commands:
   - ***aws-iam-identitycenter-create-user***
   - ***aws-iam-identitycenter-get-user***
   - ***aws-iam-identitycenter-create-group***
   - ***aws-iam-identitycenter-get-group***
```

- Added support for the following arguments in the ***aws-iam-identitycenter-list-users***, ***aws-iam-identitycenter-list-groups*** and ***aws-iam-identitycenter-list-groups-for-user*** commands.
    - *limit*
    - *nextToken*
- Added support for the following arguments in the ***aws-iam-identitycenter-create-user*** command.
    - *userType*
    - *profileUrl*
    - *title*
    - *region*

> - Added an argument *UserNextToken* to the context data in path `AWS.IAMIdentityCenter` in the ***aws-iam-identitycenter-list-users*** command.
> - Added an argument *GroupNextToken* to the context data in path `AWS.IAMIdentityCenter` in the ***aws-iam-identitycenter-list-groups*** command.
> - Added an argument *GroupsUserNextToken* to the context data in path `AWS.IAMIdentityCenter.User` in the ***aws-iam-identitycenter-list-groups-for-user*** command.

Reviewer: ```suggestion
- Added the *UserNextToken* argument to the context data in path `AWS.IAMIdentityCenter` in the ***aws-iam-identitycenter-list-users*** command.
- Added the *GroupNextToken* argument to the context data in path `AWS.IAMIdentityCenter` in the ***aws-iam-identitycenter-list-groups*** command.
- Added the *GroupsUserNextToken* argument to the context data in path `AWS.IAMIdentityCenter.User` in the ***aws-iam-identitycenter-list-groups-for-user*** command.
```

> - Deprecated the *groupName* argument and replace with the *displayName* argument in the ***aws-iam-identitycenter-get-group*** command.

Reviewer: ```suggestion
- Deprecated the *groupName* argument and replaced it with the *displayName* argument in the ***aws-iam-identitycenter-get-group*** command.
```

- Updated the following parameters to not required.
    - *identityStoreId*
    - *roleArn*
    - *roleSessionName*
    - *accessKey*
    - *secretKey*

> - Added a parameter *credentials*.

Reviewer: ```suggestion
- Added the *credentials* parameter .
```

> - Deprecated the ***aws-iam-identitycenter-remove-user-from-all-groups*** command and replaced with the ***aws-iam-identitycenter-delete-group-membership*** command.

Reviewer: ```suggestion
- Deprecated the ***aws-iam-identitycenter-remove-user-from-all-groups*** command and replaced it with the ***aws-iam-identitycenter-delete-group-membership*** command.
```

> - Added an argument *membershipId* to ***aws-iam-identitycenter-delete-group-membership*** command.
> - Updated the *emailAddress* argument to not required in ***aws-iam-identitycenter-get-user-by-email*** command.

Reviewer: ```suggestion
- Added the *membershipId* parameter to the ***aws-iam-identitycenter-delete-group-membership*** command.
- Updated the *emailAddress* argument to not required in the ***aws-iam-identitycenter-get-user-by-email*** command.
```

#### Integrations

##### AWS - IAM Identity Center - TEST

> - %%UPDATE_RN%%

Reviewer 2: Please proper release notes mention all the commands you added and edited. Also remove this line.
